.mergify.yml

```diff
@@ -0,0 +1,19 @@
+pull_request_rules:
+  - name: Automatically merge on CI success and review
+    conditions:
+      - status-success=continuous-integration/travis-ci/pr
+      - "label=ready"
```

Comment:
could we use @rz's `ready to merge` instead of just `ready`?

Reply:
yes, i actually went to every repo and updated the label to `ready to merge` but forgot to update this, thank you for catching it!